### ISSUE TITLE
Fans are louder with BIOS update

### ISSUE THREAD

**Issue by koezgen** [seerge/g-helper#294]
**BUG**
After installing the BIOS 313 update, fans became louder. 
I am referring to the default cases. Fan speeds are inconsistent with the curves in the app. 
Please refer to the SS below.

<img width="749" alt="Ekran görüntüsü 2023-04-27 170251" src="https://user-images.githubusercontent.com/95245093/234886645-32d2e30d-e392-452d-87b7-a124b042224e.png">

**Clear scenario to Reproduce**
1. Install the bios update 313 for Zephyrus G15 2022
2. Also generally, the curves do not generally match the 
3. But it got louder with this bios update.

**App Logs**
[log.txt](https://github.com/seerge/g-helper/files/11344317/log.txt)

**Desktop**
 - Windows 11
 - Zephyrus G15 2022 3060

**Asus software**
- Only the GHelper
- Asus ArmouryCrate Control Interface (should I remove this?)
- Armoury Crate is uninstalled. Only some .exe's like KeyControl. (should I remove them?)

**Comment by seerge** [comment]
@koezgen hello :) 

G15 doesn't support "reading" of fan curves at all - so app simply can't get default curve from bios. Instead of  that it shows a curve that is hardcoded in app (taken from my G14). It's needed only if you want to apply a custom fan curve and use something as base.

If "Apply custom fan curve" is not checked - curve that you see on chart doesn't play any role and bios uses whatever it has as default.

As for fan's being more "loud" (i assume they spin faster?) it's up to bios - not g-helper ;)

**Comment by koezgen** [comment]
@seerge Hello man :)
Thanks for the heads-up. I am very grateful for the tool that you provided. I did not know that this was the case with the g15. Yeah, by loud I meant RPM's. Because I did not see anything close to 2000 rpms at those temperatures before doing the bios update. I created this thread in hopes of getting this fixed in g-helper's end because a lot of people are suffering from this recent bios update. Thanks for the response.
